setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
+    description='Python client for the Yola API',
```

Comment:
> You should set long_description as @zoidbergwill described.

Oh! I misread originally and though merely having the rst files would mean they show up on pypi. Oops.

Proper long description added in https://github.com/yola/yolapy/commit/a81dac4d89b38aed4f40f3c459a1690832c43449